### PR TITLE
:hammer: Rework PPs, Use transformsV2, & Fix corruptions

### PR DESCRIPTION
**Pull request by o-laurent**
- Improve the corruption transforms
- Rework postprocessing methods
- Use transformsV2
- Fix the security issue

### CODE REVIEW

**Comment by o-laurent** [comment]
@alafage we need to state that the calibration dataloader parameters are going to be those of the validation (or test) set, then, notably for MC-BatchNorm, otherwise you wouldn't know how to set the parameters of this method. Ideally we should find a workaround because it makes no sense to define the MCBatchNorm parameter in the datamodule; maybe temporarily overwrite the dataloader's batch_size in fit?
Moreover, we should warn the user when taking the test dataset as a calibration set, and ideally provide a way to split the sets in a more principled way from the start.

**Comment by o-laurent** [comment]
I've made changes accordingly. Tell me what you think about them when you have time @alafage. Basically, I believe that the post-processing set (not only the calibration set as of now) should be handled at the datamodule level instead of the routine.

File: README.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ This package provides a multi-level API, including:
 
 - easy-to-use :zap: lightning **uncertainty-aware** training & evaluation routines for **4 tasks**: classification, probabilistic and pointwise regression, and segmentation.
 - ready-to-train baselines on research datasets, such as ImageNet and CIFAR
-- [pretrained weights](https://huggingface.co/torch-uncertainty) for these baselines on ImageNet and CIFAR ( :construction: work in progress :construction: ).
```
Comment (alafage) [inline README.md:29]: So we don’t have pretrained weights?

**Review by alafage — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (o-laurent) [reply README.md:29]: We have some, but I don't think that we can sell our library on this for now. Maybe for the submission it would be good to improve our number of pre-trained models.
